ApiBundle/Transformer/AreaTransformer.php

```diff
@@ -176,22 +188,24 @@ public function transformFromTemplate($area, TemplateInterface $template = null,
             'areaId' => $area->getAreaId()
         )));
 
-        if ($parentAreaId) {
-            $facade->addLink('_self_delete', $this->generateRoute('open_orchestra_api_area_delete_in_template_area',
-                array(
-                    'templateId' => $templateId,
-                    'parentAreaId' => $parentAreaId,
-                    'areaId' => $area->getAreaId()
-                )
-            ));
-
-        } else {
-            $facade->addLink('_self_delete', $this->generateRoute('open_orchestra_api_area_delete_in_template',
-                array(
-                    'templateId' => $templateId,
-                    'areaId' => $area->getAreaId(),
-                )
-            ));
+        if ($this->authorizationChecker->isGranted(TreeTemplatePanelStrategy::ROLE_ACCESS_DELETE_TEMPLATE)) {
```

Comment:
This transform is use for the delete an area in the node, i think you shouldn't use the access of template here. You can check ?